pgtoolkit/ctl.py

```diff
@@ -65,6 +92,32 @@ def run_command(
     )
 
 
+async def asyncio_run_command(
```

Comment:
The "asyncio" prefix is intentional because we do use asyncio as an implementation for asynchronous I/O. Users can provide, e.g., a `trio_run_command()`.